src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ApplicationTest.kt

```diff
@@ -746,10 +867,38 @@ class Cas2ApplicationTest : IntegrationTestBase() {
                     withCrn(offenderDetails.otherIds.crn)
                     withData("{}")
                     withSubmittedAt(OffsetDateTime.now().minusDays(it.toLong()))
+                    withConditionalReleaseDate(LocalDate.now().randomDateAfter())
+                  }.id,
+                )
+              }
+
+              // create 2 x submitted applications for this user
+              // with most recent first and conditional release dates of today
+              repeat(2) {
```

Comment:
quite minor but might shave off some milliseconds - do you think we need to create two each time to test that applications with CRD of today are returned?